src/BenchmarksApps/Kestrel/PlatformBenchmarks/benchmarks.plaintext.json

```diff
@@ -14,14 +14,14 @@
     "Port": 8080
   },
   "PlaintextPlatform": {
-    "Path": "/plaintext"
+    "Path": "/p"
```

Comment:
lol how is this not cheating?

Reply:
Exactly. I filed an issue on TE to clarify the requirements.

Reply:
Not invalid for current round, may be for next https://github.com/TechEmpower/FrameworkBenchmarks/issues/5582#issuecomment-609895369